sage/civilsage/templates/base.html

```diff
@@ -0,0 +1,68 @@
+{% load static %}
+{% load staticfiles %}
+
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        {% block title %}{% endblock %}
+        <meta name="viewport" content="width=device-width,
+            initial-scale=1, maximum-scale=1">
+        
+            <!-- stylesheets -->
+        <link href="{% static 'css/bootstrap.min.css' %}"
+            type="text/css" rel="stylesheet">
+        <link href="{% static 'css/custom.css' %}" type="text/css"
+            rel="stylesheet">
+        <script src="{% static 'js/jquery.min.js' %}"></script>
+        <script src="{% static 'js/bootstrap.min.js' %}"></script>
+       
+        {% block extrastyle %}{% endblock %}
+    </head>
+
+   	<body>
+        <nav class="navbar navbar-default navbar-fixed-top">
+            <div class="container">
+                <div class="navbar-header">
+            	    <button type="button" class="navbar-toggle 
+                        collapsed" data-toggle="collapse" 
+                        data-target="#navbar" aria-expanded="false" 
+                        aria-controls="navbar">
+                       <span class="sr-only">Toggle navigation</span>
+                       <span class="icon-bar"></span>
+                       <span class="icon-bar"></span>
+                       <span class="icon-bar"></span>
+                    </button>
+                    <a class="navbar-brand" href="/">
+                        Dynamic of Structure</a>
+                </div>
+                <div id="navbar" class="navbar-collapse collapse">
+                    <ul class="nav navbar-nav">
+                        <li class="active"><a href="/">Home</a></li>
+                        <li><a href="#" id="autofill"
+                            onclick="autofill()">Autofill values</a>
+                        </li>
+                    </ul>
+                    <ul class="nav navbar-nav navbar-right">
+                        <li>
+                            <a href="#" id="help-btn" 
+                                data-toggle="modal"
+                                data-target="#myModal">Help</a></li>
+                        <li>
+                            <a href="http://github.com/GreatDevelopers"
+							    target="new">About</a></li>
+                        <li class="active"><a id="help-btn" 
+                            data-toggle="modal" data-target="#contact">
+                            Contact us
+                            <span class="sr-only">(current)</span></a>
+                        </li>
+                    </ul>
+                </div><!--/.nav-collapse -->
+            </div>
+    	</nav>
+         
+        <div class="top-space"></div>
+
+        {% block body %}
```

Comment:
Also add "message" options  in base html as it is common in all html files and apply css to it.